pkg/builder/builder.go

```diff
@@ -862,20 +862,30 @@ func makeScrutinizeInitContainers(vscr *v1beta1.VerticaScrutinize, vdb *vapi.Ver
 
 // BuildVProxyDeployment builds manifest for a subclusters VProxy deployment
 func BuildVProxyDeployment(nm types.NamespacedName, vdb *vapi.VerticaDB, sc *vapi.Subcluster) *appsv1.Deployment {
+	proxyLabels := MakeLabelsForVProxyObject(vdb, sc, true)
```

Comment:
```suggestion
	proxyLabels := MakeLabelsForVProxyObject(vdb, sc, false)
```
because it is not a pod. However you probably to update `MakeCommonLabels()`, maybe add an argument `isProxy`.
If you have a better idea, feel free to implement it